lib/services/connect/middleware/security.js

```diff
@@ -16,6 +16,11 @@ function initialize(config) {
   this.config = config;
 
   if (!this.config.exclusions) this.config.exclusions = [];
+  this.__unauthorisedReasons = [
```

Comment:
it will be good to set up a enumerated and reuse it everywhere where these reasons are specified to avoid duplication and risk of only updating one instance

Reply:
I can do that, in fact there is a constants.js we can use - will make this change